extensions/GUI/CCControlExtension/CCControlButton.cpp

```diff
@@ -79,9 +79,9 @@ bool ControlButton::initWithLabelAndBackgroundSprite(Node* node, ui::Scale9Sprit
 
         _isPushed = false;
 
-        // Adjust the background image by default
-        setAdjustBackgroundImage(true);
```

Comment:
I replace "setAdjustBackgroundImage(true)"  as "setAdjustBackgroundImage(adjustBackGroundSize); " In the following  code